trusted/crypto/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "mc-sgx-crypto"
+version = "0.1.0"
+edition = "2018"
```

Comment:
In working https://github.com/mobilecoinfoundation/sgx/pull/6 I had to hold back to 2018 because `mobilecoin/rust-sgx-base:latest` is still back on a 2018 version of rust.  I may need to re-investigate and see what/why.

Reply:
NVM I'm not able to reproduce it, I must have misunderstood an earlier warning.